src/Tribe/Admin/Pages.php

```diff
@@ -0,0 +1,274 @@
+<?php
+namespace Tribe\Admin;
+
+/**
+ * Class Pages.
+ *
+ * @since TBD
+ */
+class Pages {
+	/**
+	 * Current page ID (or false if not registered with this controller).
+	 *
+	 * @since TBD
+	 *
+	 * @var string
```

Comment:
Since there is no `__construct` method in the class. there is no guarantee this property will always be a `string`.
```suggestion
	 * @var string|null
```